index.ts

```diff
@@ -5,7 +5,7 @@
 // -----------------------------------------------------------------------------
 
 import { format } from 'util';
-import clc from 'cli-color';
+import * as colorette from "colorette";
```

Comment:
```suggestion
import * as colorette from 'colorette';
```

We have been using single quotes for this file (and throughout much of the codebase); let's continue doing that.